tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java

```diff
@@ -54,49 +55,48 @@ public static WebArchive deploy() {
                 .addAsLibrary(testJar);
         return war;
     }
-    
-    @Test
+
+    @Test(priority = 1)
     public void testFallbackSuccess() {
-        
+
         try {
             String result = fallbackClient.serviceA1();
             Assert.assertTrue(result.contains("serviceA1"), "The message should be \"fallback for serviceA1\"");
             //MyBean should be injected to the fallbackA
             Assert.assertTrue(result.contains("34"), "The message should be \"fallback for serviceA1 myBean.getCount()=34\"");
         }
         catch(RuntimeException ex) {
-            Assert.fail("serviceA1 should not throw a RuntimeException in testRetryMaxRetries");
+            Assert.fail("serviceA1 should not throw a RuntimeException in testRetryMaxRetries", ex);
         }
-        
+
         try {
             String result = fallbackClient.serviceA2();
             Assert.assertTrue(result.contains("serviceA2"), "The message should be \"fallback for serviceA2\"");
-            //MyBean should be injected to the fallbackA, MyBean is application scoped, 
+            //MyBean should be injected to the fallbackA, MyBean is application scoped,
             //so the same instance should be injected to the fallback handler
             Assert.assertTrue(result.contains("35"), "The message should be \"fallback for serviceA2 myBean.getCount()=35\"");
         }
         catch(RuntimeException ex) {
             Assert.fail("serviceA2 should not throw a RuntimeException in testRetryMaxRetries");
         }
-       
+
     }
-    
+
     /**
      * The fallbakHandler is the wrong type. This should failed the validation check.
      */
-    @Test
+    @Test(priority = 2)
```

Comment:
Note that without this change, I was seeing cases where this test ran first, throwing off the counts for `testFallbackSuccess`.